components/dashboard/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    percent: number;
+};
+export const ProgressBar: FC<Props> = ({ percent }) => {
+    return (
+        <div className="w-full rounded-full h-3 my-1 bg-gray-600 dark:bg-gray-300">
```

Comment:
**thought**: Since we're using dark colors for the toast, the progress bar colors will have to be reverted if we would use the same component elsewhere, like the billing.